api-reference/10 UI Components/GridBase/1 Configuration/autoNavigateToFocusedRow.md

```diff
@@ -8,4 +8,8 @@ default: true
 Automatically scrolls to the focused row when the [focusedRowKey](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/focusedRowKey.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#focusedRowKey') is changed.
 
 ---
-<!-- Description goes here -->
+
+[note] You must specify the component's [height](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#height) to ensure that the **autoNavigateToFocusedRow** property works properly.
```

Comment:
Can you clarify what you mean by "specify the component's height" in the context of navigating to the focused row? For example, does the height need to be set to a certain minimum height, etc. for the property to work properly? Please clarify.

Reply:
DataGrid/TreeList can scroll to a specific row only using its own scroll bar. Right now, this components do not seem to have height restrictions, so the internal scrolling logic does not work. You can use the **height** option for the **autoNavigateToFocusedRow** configuration to work as intended.

Reply:
Okay, I understand now. Thank you.